docs/developer-guide/serialization.md

```diff
@@ -127,11 +129,11 @@ split into columns.
 For example, given a SQL statement such as:
 
 ```sql
-CREATE STREAM x (ID BIGINT, NAME STRING, AGE INT) WITH (VALUE_FORMAT='DELIMITED', ...);
+CREATE STREAM x (ORGID BIGINT KEY, ID BIGINT KEY, NAME STRING, AGE INT) WITH (FORMAT='DELIMITED', ...);
 ```
 
-ksqlDB splits a value of `120, bob, 49` into the three fields with `ID` of
-`120`, `NAME` of `bob` and `AGE` of `49`.
+ksqlDB splits a key of `120, 21` and a value of `bob, 49` into the four fields (two keys and two values) 
```

Comment:
Is it valid to have spaces here? I thought the default delimiter was a comma (with no space)?

Reply:
I think you are right!